Deploy-to-KServe-ModelMesh-Serving/README.md

```diff
@@ -31,7 +31,7 @@ You will need to [login to the IBM Kubernetes Service (IKS) cluster](https://clo
 ibmcloud login --sso
 
 # Set the context by updating the kubeconfig file set by KUBECONFIG environment variable, or ~/.kube/config by default.
-ibmcloud ks cluster config --cluster <iks-cluster-name>
+ibmcloud ks cluster-config --cluster <iks-cluster-name>
```

Comment:
@laurachiticariu can you check your `ibmcloud` cli and kubernetes service plugins? current versions do not use a `-` between cluster and config and will give an error.